polymesh_schema.json

```diff
@@ -1142,6 +1154,17 @@
         "PriceTier": {
             "total": "Balance",
             "price": "Balance"
+        },
+        "AssetMigrationError": {
+            "_enum": { 
+                "AssetDocumentFail": "(Ticker, DocumentId)"
+            }
+        },
+        "MigrationError": {
```

Comment:
Will this scheme scale when other pallets have problems?

Reply:
I don't think so. As soon as we add new elements, the generic type is not going to be translated to our Schema.
A common enum should be generated to wrap any new migration errors.
It is just an initial sketch, and there is a JIRA ticket where we'll cover the migration to avoid silent failures.